server/src/models/Character.js

```diff
@@ -0,0 +1,20 @@
+const Model = require("./Model.js")
+
+class Character extends Model{
+    static get tableName(){
+        return "characters"
+    }
+
+    static get jsonSchema(){
+        return {
+            type:"object",
+            required:["name", "gameTitle"],
+            properties:{
+                name: { type: "string" },
+                gameTitle: { type: "string"}
+            }
```

Comment:
We should also add our other properties to ensure they have the correct type! It looks like we have all of the required fields accounted for, but also want to make sure that other things like "series" is a string, etc